### DISCUSSION TITLE
HAL - rewrite CURIE plugin

### DISCUSSION THREAD

**Discussion by t1tcrucible** [Kong/kong#7374] [Ideas and feature requests]
In order to use APIs - using HAL/xml or HAL/json. We want to rewrite the CURIE "ea" field in the response body. This is necessary when application consumers are using the HAL links.

HAL: http://stateless.co/hal_specification.html

Is somebody already doing something like that? 

thank you

**Comment by sonicaghi** [comment]
nope. a well designed PR for this plugin is welcome! 

https://getkong.org/docs/0.5.x/plugin-development/
